### DISCUSSION TITLE
Control+E shortcut for focusing search

### DISCUSSION THREAD

**Discussion by rfeltis** [jeffvli/sonixd#385] [Ideas]
Many programs that expose a search interface have `Control+E` as a shortcut to focus the search bar. Seeing as the search button is enabled on every screen, this seems like it could be a feasible default.

**Comment by Chaphasilor** [comment]
You are aware that `ctrl + F` is the shortcut you are looking for? I've never seen `ctrl + E` for search before, only `ctrl + F` or `/`.

In the end, the best solution would probably be to allow the user to change the shortcuts to their liking :)

**Reply by rfeltis** [reply]
Agree that some type of configuration page would allow for discoverability 👍.

If you use Chrome or any browser, `ctrl + E` does a search operation. `ctrl + f` is reserved for find operations, like finding text in a page.

If you're on Windows, File Explorer responds to both `ctrl + E` and `ctrl + F`.

The latest Windows Media Player that replaced Groove Music on Windows uses `ctrl + E` to focus search. `ctrl + F` forwards to the next track.
